src/data/actions/loginForm.js

```diff
@@ -0,0 +1,63 @@
+import axios from 'axios';
+import qs from 'query-string';
+
+import {
+  LOGIN_FAILURE,
+  LOGIN_REQUEST,
+  LOGIN_SUCCESS,
+} from '../constants/loginForm';
+
+const setLoginPending = loading => ({
+  type: LOGIN_REQUEST,
+  payload: { loading },
```

Comment:
You don't need to pass the `loading` variable into this function as the `loading: true` can be set within the reducer for the `LOGIN_REQUEST` action.